fw/fe310/platform.mk

```diff
@@ -2,4 +2,4 @@ CC =     riscv32-none-elf-gcc
 AR =     riscv32-none-elf-ar
 RANLIB = riscv32-none-elf-ranlib
 
-CFLAGS = -march=rv32imac -mabi=ilp32 -mcmodel=medlow -ffunction-sections -fdata-sections --specs=nano.specs -O2
+CFLAGS = -march=rv32g -mabi=ilp32d -mcmodel=medlow -ffunction-sections -fdata-sections --specs=nano.specs -O2
```

Comment:
Any news from the author about the expected/supported extensions? We should double check that the hardware (fe310) supports double precision floating point.

Reply:
Any news on this front? If not, @leenaars may be able to help get in touch